src/twisted/application/twist/test/test_options.py

```diff
@@ -47,7 +47,7 @@ def patchOpen(self) -> None:
         """
         self.opened = []  # type: List[Tuple[str, Optional[str]]]
 
-        def fakeOpen(name: str, mode: Optional[str] = None) -> NotImplemented:
```

Comment:
this doesn't return Exception

Reply:
What is your suggestion for fixing this? :) thanks

Reply:
I wanted to suggest `Literal[NotImplemented]`, but mypy doesn't like that either:
```
error: Parameter 1 of Literal[...] is invalid  [misc]
```

Reply:
You could use `object` as the return type, that is both correct and will make mypy complain if the returned value is used as another type.